BrowserKit/Sources/ComponentLibrary/Buttons/LinkButton.swift

```diff
@@ -17,16 +19,23 @@ public class LinkButton: UIButton, ThemeApplicable {
     }
 
     public func configure(viewModel: LinkButtonViewModel) {
```

Comment:
Does changes here affects the SampleComponent app in any way? Wondering if it needs to be adjusted + can you review the docc documentation to ensure everything is up to date?

Reply:
@lmarceau no the library is fine as is, so is the docc. Although I discovered while checking that the SampleComponent app was not building so I fixed that.